gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -172,6 +172,13 @@
               </div>
             </div>
             <div class="col-md-6">
+              <div ng-if="tmpGUI.address.startsWith('/') || tmpGUI.address.startsWith('unix://') || tmpGUI.address.startsWith('unixs://')" class="form-group" ng-class="{'has-error': settingsEditor.UnixSocketPermissions.$invalid && settingsEditor.UnixSocketPermissions.$dirty}">
+                <label translate>UNIX Permissions</label>
```

Comment:
I would have expected the `Unix` kind of capitalisation", but I am not sure.

Reply:
I basically followed how it showed up already in documentation (except in the code). Let me know if you want it changed anywhere.